tests/unit/create/test_creator.py

```diff
@@ -177,8 +177,10 @@ def list_to_str(iterable):
     if sys.platform == "win32":
         exes = ("python.exe",)
     else:
-        # TODO: also test "python{}.{}" once fixed
-        exes = ("python", "python{}".format(sys.version_info[0]))
+        exes = ("python", "python{}".format(*sys.version_info), "python{}.{}".format(*sys.version_info))
+        # pypy3<=7.3: https://bitbucket.org/pypy/pypy/pull-requests/697
+        if IS_PYPY and CURRENT.pypy_version_info[:3] <= [7, 3, 0] and creator == "venv":
+            exes = exes[:-1]
```

Comment:
I am not sure this is correct. On CPython, the actual exe (what is at the base of all the symlinks) is `python/path/python3.6`, on PyPy it is `pypy/path/pypy3`. Then venv copies that base exe and creates aliases for newpath/bin/python and newpath/bin/python3. So the test should be something like

```
if IS_PYPY:
    exename = f'pypy{sys.version_info.major}'
else:
    exename = f'python{sys.version_info.major}.{sys.version_info.minor}'
exes = ("python", f'pypy{sys.version_info.major}', exename)
```

Reply:
@mattip the fact that the base is ``python3.6`` is just a matter of luck; of under what name we discover the python. It well could be any of the ``python``, ``python3`` or ``python3.6`` depending on how we discover it on the path. How it's exposed at root level from should not affect how the virtualenv makes it available 👍